### PR TITLE
Deleted broken link and relocated content

### PR DESCRIPTION
**Pull request by RichardJJG**
/assign @snneji

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,18 +6,14 @@ documentation! Check out the resources below to get started.
 - [How to contribute](https://github.com/knative/docs/tree/main/CONTRIBUTING.md) -- Contains information about how
   to contribute and outlines the roles for Docs contributors.
 
-- [Template page](https://raw.githubusercontent.com/knative/community/main/docs/new-page-template.md)
-  -- A blank documentation page that demonstrates how to format a new page and
-  includes tips on structuring your documentation.
+- Use one of the new content templates:
+   - [Documenation](https://github.com/knative/docs/tree/main/template-docs-page.md) -- Instructions and a template that
```
Comment (pmorie) [inline docs/README.md:10]: typo: documentation

**Comment by pmorie** [comment]
I'll approve this once the typo is fixed

File: docs/README.md (hunk 1 above)
Comment (rhuss) [inline docs/README.md:10]:
```suggestion
   - [Documentation](https://github.com/knative/docs/tree/main/template-docs-page.md) -- Instructions and a template that
```

File: docs/README.md [hunk 2]
```diff
@@ -6,18 +6,14 @@ documentation! Check out the resources below to get started.
...
+     you can use to help you add new documentation.
```
Comment (rhuss) [inline docs/README.md:11]:
```suggestion
     you can use for adding new documentation.
```

File: docs/README.md [hunk 3]
```diff
@@ -6,18 +6,14 @@ documentation! Check out the resources below to get started.
...
+   - [Blog](https://github.com/knative/docs/tree/main/template-blog-entry.md) -- Instructions and a template that
+     you can use to help you post to the Knative blog.
```
Comment (rhuss) [inline docs/README.md:13]:
```suggestion
     you can use for posting to the Knative blog.
```

**Comment by dprotaso** [comment]
bump @RichardJJG this needs a rebase

File: docs/README.md (hunk 1 above)
Comment (rhuss) [inline docs/README.md:10]:
```suggestion
   - [Documentation](https://github.com/knative/docs/tree/main/template-docs-page.md) -- Instructions and a template that
```

**Comment by rhuss** [comment]
ping @RichardJJG any chance to update the PR ? No worries, if not, we could rebase and apply the suggested changes.

**Comment by vaikas** [comment]
ping on this @RichardJJG would you mind rebasing and addressing the PR comments?

**Comment by RichardJJG** [comment]
I do apologize -- I don't know how I missed all these mentions. I made this small (now rather ancient) PR when I was still very confused about Git, and it fell through the cracks. And other PRs have since overlapped it. I'll close this and delete the branch; it's not worth further hassle.

**Pull request closed without merging**